docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md

```diff
@@ -0,0 +1,75 @@
+# Chart Operations 
+
+## Introduction
+
+Discover, Create, Deploy, Update, Upgrade, Delete charts.
+
+## 1. Discover the chart from the Chart Store
+
+Select the `Charts` section from the left pane, you will be landed to the `Chart Store` page. 
```

Comment:
you will land on the `Chart Store` page.

Reply:
done